apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -518,6 +518,30 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
     end
   end
 
+  defp ancestor_is_def?(env) do
+    env
+    |> cursor_path()
+    |> Enum.find_value(false, fn
+      {:def, _, _} ->
+        true
+
+      {:defp, _, _} ->
+        true
+
+      _ ->
+        false
+    end)
+  end
+
+  defp ancestor_is_type?(env) do
+    env
+    |> cursor_path()
+    |> Enum.find_value(false, fn
+      {:type, _, _} -> true
+      _ -> false
```

Comment:
same here. `any?` is clearer
```suggestion
    |> Enum.any?(fn
      {:type, _, _} -> true
      _ -> false     
```